cmd/promtool/main.go

```diff
@@ -145,6 +146,10 @@ func main() {
 	checkRulesLintFatal := checkRulesCmd.Flag(
 		"lint-fatal",
 		"Make lint errors exit with exit code 3.").Default("false").Bool()
+	checkRulesLintLookbackDelta := checkRulesCmd.Flag(
+		"lint.lookback-delta",
+		"The maximum lookback duration. This config is only used for rules linting checks.",
```

Comment:
> This config is only used for rules linting checks.

This is not true, what's actually linted is scrape configs.

Reply:
Yes, you are right. I mixed the rule lint check and the config lint check.
I fixed these related parts and moved the new check to the config lint check part.